notebooks/03-query/16-export-msq-query.ipynb

```diff
@@ -0,0 +1,426 @@
+    "sql_client = druid.sql\n",
+    "status_client = druid.status\n",
+    "\n",
+    "status_client.version"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "645f0209-436a-47e2-a1f4-be6c43de0eac",
+   "metadata": {},
+   "source": [
+    "## Load example data\n",
+    "\n",
+    "Once your Druid environment is up and running, ingest the sample data for this tutorial.\n",
+    "\n",
+    "Run the following cell to create a table called \"example-wiki-pivot-unpivot\". When completed, you'll see a description of the final table."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "032ee776-1eb9-46dc-989a-933bd7c7d1da",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "REPLACE INTO \"example-wiki-export\" OVERWRITE ALL\n",
+    "WITH \"ext\" AS (\n",
+    "  SELECT *\n",
+    "  FROM TABLE(\n",
+    "    EXTERN(\n",
+    "      '{\"type\":\"http\",\"uris\":[\"https://druid.apache.org/data/wikipedia.json.gz\"]}',\n",
+    "      '{\"type\":\"json\"}'\n",
+    "    )\n",
+    "  ) EXTEND (\"isRobot\" VARCHAR, \"channel\" VARCHAR, \"timestamp\" VARCHAR, \"flags\" VARCHAR, \"isUnpatrolled\" VARCHAR, \"page\" VARCHAR, \"diffUrl\" VARCHAR, \"added\" BIGINT, \"comment\" VARCHAR, \"commentLength\" BIGINT, \"isNew\" VARCHAR, \"isMinor\" VARCHAR, \"delta\" BIGINT, \"isAnonymous\" VARCHAR, \"user\" VARCHAR, \"deltaBucket\" BIGINT, \"deleted\" BIGINT, \"namespace\" VARCHAR, \"cityName\" VARCHAR, \"countryName\" VARCHAR, \"regionIsoCode\" VARCHAR, \"metroCode\" BIGINT, \"countryIsoCode\" VARCHAR, \"regionName\" VARCHAR)\n",
+    ")\n",
+    "SELECT\n",
+    "  TIME_PARSE(\"timestamp\") AS \"__time\",\n",
+    "  \"isRobot\",\n",
+    "  \"channel\",\n",
+    "  \"added\",\n",
+    "  \"user\",\n",
+    "  \"deleted\"\n",
+    "FROM \"ext\"\n",
+    "PARTITIONED BY DAY\n",
+    "'''\n",
+    "display_client.run_task(sql)\n",
+    "sql_client.wait_until_ready('example-wiki-export')\n",
+    "display_client.table('example-wiki-export')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "36ad757a-957e-465a-a284-855c14e3e8fd",
+   "metadata": {},
+   "source": [
+    "## Exporting data\n",
+    "\n",
+    "Exporting data uses MSQ `INSERT` to an `EXTERN` table function to define the output location and format of the data.\n",
+    "\n",
+    "The SQL statement takes on the form:\n",
+    "```\n",
+    "INSERT INTO\n",
+    "  EXTERN(<destination function>)\n",
+    "AS CSV\n",
+    "SELECT\n",
+    "  ...\n",
+    "```\n",
+    "\n",
+    "- destination function defines the output location\n",
+    "- AS CSV is the format of the exported file(s), new formats are expected as this experimental function evolves\n",
+    "- SELECT statement can be any MSQ query including transformations, joins and aggregations\n",
+    "\n",
+    "In this learning environment the `druid_export_storage_baseDir` property has been set to `/opt/shared/exports` in the docker compose \"environment\" file.\n",
```

Comment:
You could move the comment in the intro to this point:

Adjust this setting to output results to other locations, such as [S3](https://druid.apache.org/docs/latest/multi-stage-query/reference/#s3).

Reply:
done